src/Features/VisualBasic/Portable/AddImport/VisualBasicAddImportFeatureService.vb

```diff
@@ -318,7 +318,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.AddImport
             Return AddImportAsync(contextNode, document, importsStatement, options, cancellationToken)
         End Function
 
-        Private Function CreateNameSyntax(nameSpaceParts As IReadOnlyList(Of String), index As Integer) As NameSyntax
+        Private Shared Function CreateNameSyntax(nameSpaceParts As IReadOnlyList(Of String), index As Integer) As NameSyntax
```

Comment:
updated anlayzer detects recursive methods that can be made static now.  so had to fix up a bunch of new places.